src/tests/ftest/util/ec_utils.py

```diff
@@ -98,37 +119,62 @@ def ior_param_update(self, oclass, sizes):
         self.ior_cmd.dfs_dir_oclass.update(oclass[0])
         self.ior_cmd.dfs_chunk.update(sizes[0])
 
-    def ior_write_dataset(self):
+    def ior_write_single_dataset(self, oclass, sizes, percent=1):
+        """Write IOR single data set with EC object.
+
+        Args:
+            oclass(list): list of the obj class to use with IOR
+            sizes(list): Update Transfer, Chunk and Block sizes
+            percent(int): %of storage to be filled. Default it will use the
+                          given parameters in yaml file.
+        """
+        self.ior_param_update(oclass, sizes)
+
+        # Create the new container with correct redundancy factor for EC
+        self.ec_container_create(oclass[0])
+        self.update_ior_cmd_with_pool(create_cont=False)
+
+        # Start IOR Write
+        self.container.uuid = self.ec_container.uuid
+        self.start_ior_load(operation="WriteRead", percent=percent,
+                            create_cont=False)
+        self.cont_uuid.append(self.ior_cmd.dfs_cont.value)
+
+    def ior_write_dataset(self, percent=1):
         """Write IOR data set with different EC object and different sizes."""
         for oclass in self.obj_class:
             for sizes in self.ior_chu_trs_blk_size:
-                # Skip the object type if server count does not meet the minimum
-                # EC object server count
+                # Skip the object type if server count does not meet the
+                # minimum EC object server count
                 if oclass[1] > self.server_count:
                     continue
-                self.ior_param_update(oclass, sizes)
+                self.ior_write_single_dataset(oclass, sizes, percent)
 
-                # Create the new container with correct redundancy factor
-                # for EC object type
-                self.ec_container_create(oclass[0])
-                self.update_ior_cmd_with_pool(create_cont=False)
-                # Start IOR Write
-                self.container.uuid = self.ec_container.uuid
-                self.start_ior_load(operation="WriteRead", percent=1,
-                                    create_cont=False)
-                self.cont_uuid.append(self.ior_cmd.dfs_cont.value)
+    def ior_read_single_dataset(self, oclass, sizes, percent=1):
+        """Read IOR single data set with EC object.
+
+        Args:
+            oclass(list): list of the obj class to use with IOR
+            sizes(list): Update Transfer, Chunk and Block sizes
+            percent(int): %of storage to be filled. Default it will use the
+                          given parameters in yaml file
+        """
+        self.ior_param_update(oclass, sizes)

```

Comment:
(style)  DOS line endings
(pylint-mixed-line-endings) Mixed line endings LF and CRLF